jgrapht-demo/src/main/java/org/jgrapht/demo/LabeledEdges.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * (C) Copyright 2012-2018, by Oliver Schrenk and Contributors.
```

Comment:
It seems I was mistaken. Can you change the copyright of this class to Barak Naveh